{"kind":"symbol","hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason"]]},"identifier":{"url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonInitial","interfaceLanguage":"occ"},"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["occ"],"tokens":[{"text":"SMIChallengeReasonInitial","kind":"identifier"}],"platforms":["iOS"]}]}],"metadata":{"externalID":"c:@E@SMIChallengeReason@SMIChallengeReasonInitial","navigatorTitle":[{"text":"SMIChallengeReasonInitial","kind":"identifier"}],"roleHeading":"Enumeration Case","modules":[{"name":"SMIClientCore"}],"title":"SMIChallengeReasonInitial","symbolKind":"case","role":"symbol","fragments":[{"text":"SMIChallengeReasonInitial","kind":"identifier"}]},"abstract":[{"text":"The initial challenge for an user verification token.","type":"text"}],"variants":[{"paths":["\/documentation\/smiclientcore\/smichallengereason\/smichallengereasoninitial"],"traits":[{"interfaceLanguage":"occ"}]}],"schemaVersion":{"minor":3,"major":0,"patch":0},"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason/SMIChallengeReasonInitial":{"title":"SMIChallengeReasonInitial","abstract":[{"text":"The initial challenge for an user verification token.","type":"text"}],"fragments":[{"text":"SMIChallengeReasonInitial","kind":"identifier"}],"kind":"symbol","type":"topic","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonInitial","url":"\/documentation\/smiclientcore\/smichallengereason\/smichallengereasoninitial","role":"symbol"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"kind":"symbol","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","type":"topic","role":"collection","title":"SMIClientCore","url":"\/documentation\/smiclientcore","abstract":[]},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason":{"title":"SMIChallengeReason","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason","kind":"symbol","url":"\/documentation\/smiclientcore\/smichallengereason","type":"topic","fragments":[{"kind":"identifier","text":"SMIChallengeReason"}],"abstract":[{"text":"The reason for a credential challenge. Possible values:","type":"text"}],"role":"symbol"}}}